oriedita/src/main/resources/name.properties

```diff
@@ -1,222 +1,222 @@
+o_F_checkAction=Check Errors in Polygon
+foldableLinePlusGridInputAction= Flat Foldable Line (extend)
+select_polygonAction=Select Polygon
+unselect_polygonAction=Deselect Polygon
+select_lXAction=Select Intersected Lines
+unselect_lXAction=Deselect Intersected Lines
+del_lAction=Delete Included Lines
+del_l_XAction=Delete Intersected Lines
+selectAnd3ClickAction=Select and 3 Click Action
+replace_lineAction=Paint Bucket
+switchReplaceAction=Swap Line Types
 axiom5Action=Axiom 5
 axiom7Action=Axiom 7
 # Grid configure
 gridConfigureOkAction=
-gridSizeDecreaseAction=Decrease grid size
-gridSizeSetAction=Set the grid size
-gridSizeIncreaseAction=Increase grid size
-gridColorAction=Set the grid color
-gridLineWidthDecreaseAction=Decrease grid line width
-gridLineWidthIncreaseAction=Increase grid line width
-changeGridStateAction=Change grid state paper/full/hide
-moveIntervalGridVerticalAction=Move interval grid vertically
-setIntervalGridSizeAction=Set interval grid size
-moveIntervalGridHorizontalAction=Move interval grid horizontally
-intervalGridColorAction=Set interval grid color
-setGridParametersAction=Apply grid changes
-gridConfigureResetAction=Reset grid
+gridSizeDecreaseAction=Decrease Grid Size
+gridSizeSetAction=Set Grid Size
+gridSizeIncreaseAction=Increase Grid Size
+gridColorAction=Grid Color
+gridLineWidthDecreaseAction=Decrease Grid Line Width
+gridLineWidthIncreaseAction=Increase Grid Line Width
+changeGridStateAction=Toggle Grid
+moveIntervalGridVerticalAction=Move Interval Vertically
+setIntervalGridSizeAction=Set Interval Size
+moveIntervalGridHorizontalAction=Move Interval Horizontally
+intervalGridColorAction=Interval Color
+setGridParametersAction=Apply Grid Changes
+gridConfigureResetAction=Reset Grid
 # Folded Figure Resize
-foldedFigureSizeSetAction=Set size of folded figure
-foldedFigureSizeDecreaseAction=Decrease size of folded figure
-foldedFigureSizeIncreaseAction=Increase size of folded figure
+foldedFigureSizeSetAction=Set Fold Size
+foldedFigureSizeDecreaseAction=Decrease Fold Size
+foldedFigureSizeIncreaseAction=Increase Fold Size
 # Folded Figure Rotate
-foldedFigureRotateAntiClockwiseAction=Rotate folded figure anti-clockwise
-foldedFigureRotateSetAction=Set rotation of folded figure
-foldedFigureRotateClockwiseAction=Rotate folded figure clockwise
+foldedFigureRotateAntiClockwiseAction=Rotate Fold Anti-clockwise
+foldedFigureRotateSetAction=Set Rotation of Fold
+foldedFigureRotateClockwiseAction=Rotate Fold Clockwise
 # Menu
-newAction=_New
-openAction=_Open...
-saveAction=_Save
-saveAsAction=Save _as...
-importAction=_Import
-exportAction=_Export
-exitAction=E_xit
-showPointRangeAction=
-pointOffsetAction=
-gridInputAssistAction=
-displayCommentsAction=
-displayCpLinesAction=
-displayAuxLinesAction=
-displayLiveAuxLinesAction=
-displayStandardFaceMarksAction=
-cpOnTopAction=
-toggleHelpAction=
-textAction=Add/Edit/Delete Text
\ No newline at end of file
+newAction=New
```

Comment:
_New makes the N the navigation key in the menu

Reply:
Ah right thats what they do, I'll add them back